README.md

```diff
@@ -568,37 +569,143 @@ It's important to understand the following points:
 - dbt does not manage lf-tags for database
 - dbt does not manage lakeformation permissions
 
-That's why you should handle this by yourself manually or using some automation tools like terraform, AWS CDK etc.  
+That's why you should handle this by yourself manually or using some automation tools like terraform, AWS CDK etc.
 You may find the following links useful to manage that:
 
 <!-- markdownlint-disable -->
 * [terraform aws_lakeformation_permissions](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lakeformation_permissions)
 * [terraform aws_lakeformation_resource_lf_tags](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lakeformation_resource_lf_tags)
 <!-- markdownlint-restore -->
 
+### Working example
```

Comment:
Working examples should refer to snapshots, but it's put after AWS Lakeformation section. Could you please fix it?

Reply:
Done in 629da5e!